closure/templates/closure_js_template_library.bzl

```diff
@@ -69,7 +79,13 @@ _closure_js_template_library = rule(
         ),
         "outputs": attr.output_list(),
         "globals": attr.label(allow_single_file = True),
-        "plugin_modules": attr.label_list(),
+        "plugin_modules": attr.string_list(
```

Comment:
This is a breaking change.
TBH, I don‘t see how the old version could ever possibly work (if it even did). I‘ll need to think about that for a while and see whether I can figure out how this was supposed to work.

Reply:
I'm under the strong impression that the old version never worked, so I didn't consider this a breaking change.